src/RefreshButton.scss

```diff
@@ -1,15 +1,57 @@
 @import 'variables';
 
-.RefreshButton-svg {
-    width: 1.4rem;
-    height: 1.4rem;
-}
-
 .RefreshButton svg path {
-    fill: black;
     stroke: none;
 }
 
+.RefreshButton {
+    background-color: inherit;
+    margin: auto 0;
+
+    svg {
+        path {
+            fill: black;
+        }
+        path.inner-circle {
+            fill: #EA7362;
+        }
+    }
+
+    &:focus {
+        box-shadow: 0 0 0 $c2lc-focus-indicator-width $c2lc-focus-indicator-color;
```

Comment:
The colour we are using for c2lc-focus-indicator-color is #000000, and I think we should update that variable to also use #1E1E1E.

Reply:
Fixed.